### PR TITLE
Little additions to cargo sec! (Now with AMR removed due to Trillby)

### PR DESCRIPTION
**Pull request by Chevalt**
So, I have added options to buy one anti-material rifle, one RPG, and some STS 7.5 rifles at cargo, they're accurately priced as to not be overpowered, although those may be liable to change in case it's not enough.

### CODE REVIEW

File: code/modules/cargo/packs_security.dm [hunk 1]
```diff
@@ -408,14 +419,46 @@
...
+	
+	
+/datum/supply_pack/voidwolfantimaterial
+	name = "Void Wolf Mecha Hunter Kit"
+	contains = list(/obj/item/gun/projectile/boltgun/heavysniper,
+					/obj/item/ammo_magazine/ammobox/antim,
+					/obj/item/ammo_magazine/ammobox/antim,
+					/obj/item/gun_upgrade/barrel/gauss)
+					
+	cost = 4000
```
Comment (Trilbyspaceclone) [inline code/modules/cargo/packs_security.dm:439]: An ARM for 4000~ credits it way to low...

File: code/modules/cargo/packs_security.dm [hunk 2]
```diff
@@ -408,14 +419,46 @@
 	
 	
 /datum/supply_pack/voidwolfmarksman
-	name = "Void Marksman Kit"
+	name = "Void Wolf Marksman Kit"
 	contains = list(/obj/item/gun/projectile/boltgun/scout,
 					/obj/item/ammo_magazine/ammobox/heavy_rifle_408_small/hv,
 					/obj/item/gun_upgrade/barrel/gauss)
 					
 	cost = 1600
```
[the remark this answers is not in the document]
Comment (Chevalt) [reply code/modules/cargo/packs_security.dm:427]: 2500 sounds like a good price for a 408 bolt action. putting it at 3600 would be too high compared to its market price

**Comment by cdb-is-not-good** [comment]
>5500 for an RPG and a buncha rockets
>accurately priced

My dude, no. Also hard no on putting easy to access to EXPLOSIVES in cargo that anyone can order

File: code/modules/cargo/packs_security.dm [hunk 3]
```diff
@@ -408,14 +419,33 @@
 	
 	
 /datum/supply_pack/voidwolfmarksman
-	name = "Void Marksman Kit"
+	name = "Void Wolf Marksman Kit"
 	contains = list(/obj/item/gun/projectile/boltgun/scout,
 					/obj/item/ammo_magazine/ammobox/heavy_rifle_408_small/hv,
```
Comment (Trilbyspaceclone) [inline code/modules/cargo/packs_security.dm:424]:
```suggestion
					/obj/item/ammo_magazine/ammobox/heavy_rifle_408_small,
```

**Comment by Chevalt** [comment]
> > 5500 for an RPG and a buncha rockets
> > accurately priced
> 
> My dude, no. Also hard no on putting easy to access to EXPLOSIVES in cargo that anyone can order
> 
> Also another hard no on ordering SLAP. We've made that a guild exclusive
> 
> Also why is the STS crate only 3k? The rifles are 1650 EACH.

The base AKM crate is 2000 when they have 4 rifles, more or less equivalent to the STS, I'd say a thousand more than that is very much alright for weapons that are better, but not THAT much better, and delivered without ammunition, whilst having less guns.

**Comment by SouthyOsmanli** [comment]
> The base AKM crate is 2000 when they have 4 rifles, more or less equivalent to the STS, I'd say a thousand more than that is very much alright for weapons that are better, but not THAT much better, and delivered without ammunition, whilst having less guns.

Keep in mind - with the new trade beacon some guns are bought back. I have to double-check, but I /believe/ you can-resell the STS-30s for the money back closer to market value.

**Comment by cdb-is-not-good** [comment]
> > > 5500 for an RPG and a buncha rockets
> > > accurately priced
> > 
> > 
> > My dude, no. Also hard no on putting easy to access to EXPLOSIVES in cargo that anyone can order
> > Also another hard no on ordering SLAP. We've made that a guild exclusive
> > Also why is the STS crate only 3k? The rifles are 1650 EACH.
> 
> The base AKM crate is 2000 when they have 4 rifles, more or less equivalent to the STS, I'd say a thousand more than that is very much alright for weapons that are better, but not THAT much better, and delivered without ammunition, whilst having less guns. Not to mention the HV box for the bolt action rifle is very small, it's not the large SLAP boxes of 150 rounds, but those with 30 or 20 rounds (I forgot how much was it for the 408)

the AK crate is relatively not great, but we'll ignore that point for now. As Rebel pointed out yes, you can just straight up resell them back and even if not? the price is WAY too low for four relatively high end rifles. I realize that it's a 30 round box, but that's still a NO. We've gone far out of our way to NOT have HV made accessible outside of the guild.

**Comment by Chevalt** [comment]
> > The base AKM crate is 2000 when they have 4 rifles, more or less equivalent to the STS, I'd say a thousand more than that is very much alright for weapons that are better, but not THAT much better, and delivered without ammunition, whilst having less guns.
> 
> Keep in mind - with the new trade beacon some guns are bought back. I have to double-check, but I /believe/ you can-resell the STS-30s for the money back closer to market value. That's just my main input on this. I think the rockets should also have a price-hike since I added new EMP rounds to them too, they are relatively powerful - their issue is that sometimes direct hits won't count as a direct hit damage-wise. Quirky issue.

Okay, but if we accurately price guns on cargo, the AKM crate is gonna be 5000, the STS crate is gonna be 8000, who is gonna buy that outside of department leaders? No one. It's already long enough (approximately 1 to 2 hours of scavenging) for a regular prospector to buy around 5000's worth of crates, so unless we start marketing single-gun crates, this isn't just viable

**Comment by cdb-is-not-good** [comment]
> > > The base AKM crate is 2000 when they have 4 rifles, more or less equivalent to the STS, I'd say a thousand more than that is very much alright for weapons that are better, but not THAT much better, and delivered without ammunition, whilst having less guns.
> > 
> > 
> > Keep in mind - with the new trade beacon some guns are bought back. I have to double-check, but I /believe/ you can-resell the STS-30s for the money back closer to market value. That's just my main input on this. I think the rockets should also have a price-hike since I added new EMP rounds to them too, they are relatively powerful - their issue is that sometimes direct hits won't count as a direct hit damage-wise. Quirky issue.
> 
> Okay, but if we accurately price guns on cargo, the AKM crate is gonna be 5000, the STS crate is gonna be 8000, who is gonna buy that outside of department leaders? No one. It's already long enough (approximately 1 to 2 hours of scavenging) for a regular prospector to buy around 5000's worth of crates, so unless we start marketing single-gun crates, this isn't just viable

Ah yes, all or nothing. Or maybe just raise the price a bit to be a bit more sane? Scavs already round-start empty out their account for gear, you say "no one will buy it" but, my dude? Yes. They will.

**Comment by Chevalt** [comment]
> > > > The base AKM crate is 2000 when they have 4 rifles, more or less equivalent to the STS, I'd say a thousand more than that is very much alright for weapons that are better, but not THAT much better, and delivered without ammunition, whilst having less guns.
> > > 
> > > 
> > > Keep in mind - with the new trade beacon some guns are bought back. I have to double-check, but I /believe/ you can-resell the STS-30s for the money back closer to market value. That's just my main input on this. I think the rockets should also have a price-hike since I added new EMP rounds to them too, they are relatively powerful - their issue is that sometimes direct hits won't count as a direct hit damage-wise. Quirky issue.
> > 
> > 
> > Okay, but if we accurately price guns on cargo, the AKM crate is gonna be 5000, the STS crate is gonna be 8000, who is gonna buy that outside of department leaders? No one. It's already long enough (approximately 1 to 2 hours of scavenging) for a regular prospector to buy around 5000's worth of crates, so unless we start marketing single-gun crates, this isn't just viable
> 
> Ah yes, all or nothing. Or maybe just raise the price a bit to be a bit more sane? Scavs already round-start empty out their account for gear, you say "no one will buy it" but, my dude? Yes. They will.

They start with at maximum 1000 in their account and at minimum 200 or 300, that's not exactly a fortune. Now, if the STS are accurately priced (1650 per unit) the crate for 2 rifles is gonna be 3300, the crate for 4 rifles is gonna be 6600. That's a lot, that's more than the actual worth of most very, very high tier weapons in the game, like the AMR (4000 of value in code), to me it's better if we price stuff so that regular people can go to cargo in the first one or two hours in the game to lessen the already very heavy grind of some departments.

**Comment by Kazkin** [comment]
I'm gonna leave this open. I'm unconvinced that we should make RPG's so easily obtained but at the same time... it really isn't that hard to blow shit up on our server. Getting an equal explosive or greater could be done more easily by multiple jobs. I'll think about it and let more weigh in for now.

**Comment by lolman360** [comment]
rpgs in cargo is probably fine, considering they're about the same tier (but slightly more niche) than fenrirs, which are absurdly common (which is more an issue with fenrirs being in the loot pool at all but that's out of scope)

**Comment by cdb-is-not-good** [comment]
> rpgs in cargo is probably fine, considering they're about the same tier (but slightly more niche) than fenrirs, which are absurdly common (which is more an issue with fenrirs being in the loot pool at all but that's out of scope)

What? "Absurdly common"?? There's an average of like one vault per gauntlet or maybe two. It can be higher but usually isn't, and per that total number of vaults there's not often more than 2 MAYBE three fenrirs MAX, plus a possible armory spawn(never seen more than one).

**Comment by BigBadPossum** [comment]
Still a tough call on this, gonna let it sit for a bit longer in case anyone wants to weigh in.

**Comment by Dopiotl** [comment]
If cargonia gets more dangerous stuff, I wish cargonia (cagonia players) gets more cruel consequences for selling dangerous stuff. A liability clause in the law, not just the SoP. Something a marshal can and will and must follow a given charge from a customer, to the seller. That is, for specific items, namely AMR's and RPG's has the potential to gib people, kill them in one hit, remove them from the round.

It needs to stay dangerous and exiting and FUN for everyone. At least AMR's and RPG's shouldn't defcon the server...! right?

From my perspective, it seems to me we do not want unmonitored distribution of that kind of item at cargo.

I kinda see it like how we've put in place in-game laws to arrest and remove characters without records, giving into a roleplay situation for new players, and leaving staff's hands free of that boring stuff. It's a technicality, maybe a bit of a cruel but needed condition. Of course, you /can/ still play without records, just don't get caught.

So, an item watch list. I think it wouldn't be out of the question that RPG's and AMR's at cargo ought to have a special label/serial number, must stay traceable to it's SELLER and OWNER at all times, at the request of any diligent marshal officer.

Falsification of that information would make the item major contraband, and should bring at least an orange charge to both the seller/customer, a cruel but needed condition. The purpose is to force the seller to not sell those items to anyone willy nilly, and the customer is going to need to build a proper trust relation with the seller. This kind of item needs to be treated as an exception and not be normalised. 

(As an aside: as it stands right now, with how fenrirs have become so common, I'd like those too to be on a weapons watch list, if only for roleplay - maybe even some of the bluecross items too like the not-excelsior maxim)

On the same topic, a supermatter would also lend to this item watch list, since plasma fire has the potential to end the round (and has two or three times, to the point possum made a ruling to outright ban people for a month with no appeal next time it happens.)

Otherwise uh. That kind of weapon powercreep at cargo with no ic/ooc optics and rulings? I'd give it a hard no.

**Comment by Dopiotl** [comment]
In fact, I just realised this watch list system is already in full effect for the recent genetics stuff. All of the genetic mutations given to people must be approved, stamped, documented, and copied, and then furnished on paper to marshals. So...

**Comment by BigBadPossum** [comment]
After thinking it over, ultimately I do not think it would be a good addition, sadly.

**Pull request closed without merging**